test/arc_txt_endpoint_test.go

```diff
@@ -237,3 +238,174 @@ func TestHttpPost(t *testing.T) {
+	jsonPayload := fmt.Sprintf(`{"rawTx": "%s"}`, txHexString)
+	resp, err := postTx(t, jsonPayload, nil)
+	if err != nil {
```

Comment:
It's probably easer to use https://pkg.go.dev/github.com/stretchr/testify/require
like this:
``` require.NoError(t, err)
```

Reply:
In this case you could use `require.NoErrorf(t, err, "Error sending HTTP request")`

Reply:
yup, that's most thoughtful way. Thanks @farzan1990